workers/unity/Packages/com.improbable.gdk.core/Utility/CommandLineUtility.cs

```diff
@@ -5,6 +5,36 @@ namespace Improbable.Gdk.Core
 {
     public static class CommandLineUtility
     {
+        public static Dictionary<string, string> GetArguments()
+        {
+#if UNITY_ANDROID
```

Comment:
don't like to have that inside Core. we tried to move everything mobile related to its own package. Let's have a AdbPackage in the Android part of the mobile package

Reply:
This is pretty standard to have? I really don't want to do even more packages just to avoid a standard C# flow

Reply:
we already have a mobile package?

Reply:
But we also already have this CommandLineUtility class for all come that works with the command line and parsing it. I don't want this to great a dependency on the mobile package.